libraries/elixir/mix.exs

```diff
@@ -0,0 +1,48 @@
+defmodule StandardWebhooks.MixProject do
```

Comment:
It is missing some of the required fields to publish the package to hex. https://hex.pm/docs/publish#example-mixexs-file